main.go

```diff
@@ -529,9 +530,36 @@ func main() {
 		}
 	}
 
+	var configFiles []string
+
+	if opts.targetKubeConfig != "" {
+		configFiles = append(configFiles, opts.targetKubeConfig)
+
+		if targetK8sConfig.TLSClientConfig.CertFile != "" {
+			configFiles = append(configFiles, targetK8sConfig.TLSClientConfig.CertFile)
+		}
+	}
+
+	hubCfg, err := clientcmd.BuildConfigFromFlags("", opts.hubConfigPath)
+	if err != nil {
+		log.Error(err, "Could not load hub config")
+	} else {
+		configFiles = append(configFiles, opts.hubConfigPath)
+
+		if hubCfg.TLSClientConfig.CertFile != "" {
+			configFiles = append(configFiles, hubCfg.TLSClientConfig.CertFile)
+		}
+	}
+
+	configChecker, err := addonutils.NewConfigChecker("config-policy-controller", configFiles...)
```

Comment:
Could you please move this to below the `if opts.enableLease {` if statement so that we can reuse `hubCfg` that gets loaded when starting the lease?